launcher/ui/pages/instance/ServersPage.cpp

```diff
@@ -433,6 +481,16 @@ class ServersModel : public QAbstractListModel {
         }
     }
 
+    void queryServersStatus()
+    {
+        ConcurrentTask::Ptr job(new ConcurrentTask("Query servers status", APPLICATION->settings()->get("NumberOfConcurrentTasks").toInt()));
+        for (auto& server : m_servers) {
+            ServerPingTask *task = new ServerPingTask(this, server);
+            job->addTask(Task::Ptr(task));
+        }
+        job->start();
```

Comment:
This looks like if  is not blocking the task will not be executed as `job` will be destructed